assets/js/googlesitekit/datastore/user/key-metrics.js

```diff
@@ -476,8 +478,14 @@ const baseSelectors = {
 						return false;
 					}
 
-					if ( widget.displayInList ) {
-						return widget.displayInList( select, isViewOnly );
+					if (
```

Comment:
Hey @jimmymadon, I just want to flag my comment in [Slack](https://fueled10up.slack.com/archives/CBKKQEBR9/p1729596302338819?thread_ts=1728663109.141029&cid=CBKKQEBR9) about this fix not working for the "Most popular products" tile's case, so it doesn't get lost.

Cc @tofumatt

Reply:
Ah thanks for flagging this @techanvil. I thought I tested this by selecting that tile but it only renders when there is a product post type. I will add this to #9548 and make this issue a P0 bug and reproduce it there. So then I can get some ACs and an IB for it soon and fix it properly rather than issue another quick fix here. It seems that no one has surprisingly complained as yet - so not many people might be using this tile.

Reply:
Thanks @jimmymadon! Fixing the whole thing properly in #9548 sounds good. It's a good thing the error case is tucked away a bit - not only do users have to be using the tile, but also sharing the dashboard and using it in a view-only context. So, yeah, it doesn't look like many/any people are travelling this path yet, thankfully!